---
layout: page
title: Publications
page_title: Publications
permalink: /publications/
---
 (See also <a href="https://scholar.google.com/citations?user=Vff5EiwAAAAJ" target="_blank">Google Scholar</a>; # for corresponding authors; bold font for lab members)

### Computational methodologies for genome assembly

- <strong>Cheng H</strong>, Asri M, Lucas J, Koren S, Li H#. "Scalable telomere-to-telomere assembly for diploid and polyploid genomes with double graph." <a href="https://www.nature.com/articles/s41592-024-02269-8" target="_blank"><strong><em>Nat Methods</em></strong></a> (2024).

- Larivière D, Abueg L, Brajuka N, Gallardo-Alba C, Grüning B, …, <strong>Cheng H</strong>, …, Formenti G#. "Scalable, accessible and reproducible reference genome assembly and evaluation in Galaxy." <a href="https://www.nature.com/articles/s41587-023-02100-3" target="_blank"><strong><em>Nat Biotechnol</em></strong></a> (2024).

- <strong>Cheng H</strong>, Jarvis ED, Fedrigo O, Koepfli KP, Urban L, Gemmell NJ, Li H#. "Haplotype-resolved assembly of diploid genomes without parental data." <a href="https://www.nature.com/articles/s41587-022-01261-x" target="_blank"><strong><em>Nat Biotechnol</em></strong></a> (2022).

- Feng X, <strong>Cheng H</strong>, Portik D. and Li H#. "Metagenome assembly of high-fidelity long reads with hifiasm-meta." <a href="https://www.nature.com/articles/s41592-022-01478-3" target="_blank"><strong><em>Nat Methods</em></strong></a> (2022).

- <strong>Cheng H</strong>, Concepcion GT, Feng X, Zhang H, Li H#. "Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm." <a href="https://www.nature.com/articles/s41592-020-01056-5" target="_blank"><strong><em>Nat Methods</em></strong></a> (2021).

### Large-scale genome analysis

- Liao WW, Asri M, Ebler J, Doerr D, Haukness M, …, <strong>Cheng H</strong>, …, Paten B#. "A draft human pangenome reference." <a href="https://www.nature.com/articles/s41586-023-05896-x" target="_blank"><strong><em>Nature</em></strong></a> (2023).

- Jarvis ED, Formenti G, Rhie A, Guarracino A, Yang C, …, <strong>Cheng H</strong>, …, Miga KH#. "Semi-automated assembly of high-quality diploid human reference genomes." <a href="https://www.nature.com/articles/s41586-022-05325-5" target="_blank"><strong><em>Nature</em></strong></a> (2022).

- Nurk S, Koren S, Rhie A, Rautiainen M, Bzikadze AV, …, <strong>Cheng H</strong>, …, Phillippy AM#. "The complete sequence of a human genome." <a href="https://www.science.org/doi/10.1126/science.abj6987" target="_blank"><strong><em>Science</em></strong></a> (2022).

- Wagner J, Olson ND, Harris L, McDaniel J, <strong>Cheng H</strong>, …, Sedlazeck FJ#. "Curated variation benchmarks for challenging medically relevant autosomal genes." <a href="https://www.nature.com/articles/s41587-021-01158-1" target="_blank"><strong><em>Nat Biotechnol</em></strong></a> (2022).

### String algorithms for sequence analysis

- Zhang Y, Chu J, <strong>Cheng H</strong>, Li H#. "De novo reconstruction of satellite repeat units from sequence data." <a href="https://genome.cshlp.org/content/33/11/1994" target="_blank"><strong><em>Genome Res</em></strong></a> (2023).

- Zhang H, Song L, Wang X, <strong>Cheng H</strong>, Wang C, ..., Li H#. "Fast alignment and preprocessing of chromatin profiles with Chromap." <a href="https://www.nature.com/articles/s41467-021-26865-w" target="_blank"><strong><em>Nat Commun</em></strong></a> (2021).

- Zhang H, Li H, Jain C, <strong>Cheng H</strong>, Au KF, Li H, Aluru S#. "Real-time mapping of nanopore raw signals." <a href="https://academic.oup.com/bioinformatics/article/37/Supplement_1/i477/6319675" target="_blank"><strong><em>Bioinformatics</em></strong></a> (2021).

- <strong>Cheng H</strong>, Zhang Y, Xu Y#. "BitMapper2: A GPU-Accelerated All-Mapper Based on the Sparse q-Gram Index." <a href="https://ieeexplore.ieee.org/document/8330017" target="_blank"><strong><em>IEEE/ACM Trans Comput Biol Bioinform</em></strong></a> (2018).

- <strong>Cheng H</strong>, Wu M, Xu Y#. "FMtree: a fast locating algorithm of FM-indexes for genomic data." <a href="https://academic.oup.com/bioinformatics/article/34/3/416/4160683" target="_blank"><strong><em>Bioinformatics</em></strong></a> (2018).

- <strong>Cheng H</strong>, Jiang H, Yang J, Xu Y, Shang Y#. "BitMapper: an efficient all-mapper based on bit-vector computing." <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0626-9" target="_blank"><strong><em>BMC Bioinformatics</em></strong></a> (2015).

